subseas_flex_fcst/maproom.py

```diff
@@ -130,55 +130,81 @@ def write_mapTitle(startDate, leadTime, leadTimeOptions):
     targetPeriod = leadTimeOptions.get(leadTime)
     return f'{targetPeriod} {CONFIG["variable"]} Forecast issued {startDate}'
 
+
+def sel_center(spatial_array, lat, lng, dim_y="Y", dim_x="X"):
+    half_res = (spatial_array[dim_y][1] - spatial_array[dim_x][0]) / 2
+    tol = np.sqrt(2 * np.square(half_res)).values
+    return spatial_array.sel(method="nearest", tolerance=tol, **{dim_x:lng, dim_y:lat})
```

Comment:
The function looks good, I would just change the name. I thought "sel_center" was going to be a function that finds the center of the spatial array, i.e. the operation you're doing in layout.py line 42. Call it sel_snap? 

This is one of those things that should be built in to xarray, but isn't because it doesn't mange cell bounds. It would be interesting to see if xCDAT implements this operation.